### ISSUE TITLE
NTR: Cystathionine present in CSF (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8498]
New term request Cystathionine present in CSF (segal_030502141348): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000280
- label: Elevated CSF cystathionine concentration
- parents: [Abnormal CSF amino acid concentration (HP:0500184)](https://hpo.jax.org/app/browse/term/HP:0500184)
- pmid: 
- def: Concentration of cystathionine in the cerebrospinal fluid (CSF) above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8498
- synonyms:
